examples/src/main/res/values/styles.xml

```diff
@@ -8,4 +8,13 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="InstructionView">
```

Comment:
@Guardiola31337 - please refer this for how developers can customize each UI components.
Developers can define their own style and apply the style to the Mapbox UI components. For dark theme support, there are several ways to do it. Here I use the way that define dark mode colors for each attributes. So it will naturally apply to the view when dark mode enabled.